Mage/Command/Factory.php

```diff
@@ -43,14 +43,17 @@ public static function get($commandName, Config $config)
             // try a custom command
             $className = 'Command\\' . $commandName;
 
+            // TODO use a custom exception
             if (!class_exists($className)) {
                 throw new Exception('Command "' . $commandName . '" not found.');
             }
         }
 
         /** @var AbstractCommand $instance */
+        // TODO dependencies like $config should be injected into constructor
```

Comment:
Can you please link to that issue? I don't understand why you're referring to pear. Do you mean `Config` when you write about external dependencies?